engines/advancedDetector.cpp

```diff
@@ -327,34 +328,57 @@ Common::Error AdvancedMetaEngine::createInstance(OSystem *syst, Engine **engine)
 
 void AdvancedMetaEngine::reportUnknown(const Common::FSNode &path, const ADFilePropertiesMap &filesProps, const ADGameIdList &matchedGameIds) const {
 	Common::String report = Common::String::format(
+			"The game in '%s' seems to be an unknown %s engine game "
+			"variant.\n\nPlease report the following data to the ScummVM "
+			"team at %s along with the name of the game you tried to add and "
+			"its version, language, etc.:",
+			path.getPath().c_str(), getName(), "https://bugs.scummvm.org/");
+	
+	Common::String reportTranslated = Common::String::format(
```

Comment:
You can avoid duplication by having the format string in a separate variable.

Reply:
I don't know if this could be avoided since the message contains strings that could not be translated if I simply tried to do something like reportTranslated = _(report)...

Reply:
I am talking about the format string only, something along the lines of:

    const char *reportFormat = Common::String("The game in '%s' ...");
    Common::String report           = Common::String::format(  reportFormat,  ...);
    Common::String reportTranslated = Common::String::format(_(reportFormat), ...)

Reply:
Thanks! Fixed.